test/test-platform/TestPlatform.cpp

```diff
@@ -291,7 +291,7 @@ bool CTestPlatform::createExclusiveSelectionCriterionFromStateList(
 
         const std::string& strValue = remoteCommand.getArgument(state + 1);
 
-        if (!pCriterionType->addValuePair(state, strValue, strResult)) {
+        if (!pCriterionType->addValuePair(int(state), strValue, strResult)) {
```

Comment:
- maybe `state` should not be an `size_t` but rather an int ? Either case an overflow check might be useful. Although maybe just add a `TODO` as test-platform is a test executable (and `nbState` < `argc` if using remote-process).